ts/webui/src/static/model/experiment.ts

```diff
@@ -82,10 +98,13 @@ class Experiment {
     public async update(): Promise<boolean> {
         let updated = false;
 
-        await requestAxios(`${MANAGER_IP}/experiment`)
-            .then(data => {
-                updated = updated || !compareProfiles(this.profileField, data);
-                this.profileField = data;
+        await Promise.all([requestAxios(`${MANAGER_IP}/experiment`), requestAxios(`${MANAGER_IP}/experiment-metadata`)])
+            .then(([profile, metadata]) => {
+                updated ||= !compareProfiles(this.profileField, profile);
+                this.profileField = profile;
+
+                updated ||= JSON.stringify(this.metadataField) !== JSON.stringify(metadata);
```

Comment:
need to update page when metadata update?

Reply:
I'm not sure. Removing it for now.

Reply:
OK. I could test it during the bug bash.